ext/spl/spl_array.c

```diff
@@ -89,6 +89,21 @@ static inline HashTable **spl_array_get_hash_table_ptr(spl_array_object* intern)
 }
 /* }}} */
 
+static void spl_array_illegal_offset(const zval *offset)
+{
+	zend_type_error("Cannot access offset of type %s on ArrayObject", zend_get_type_by_const(Z_TYPE_P(offset)));
+}
+
+static void spl_array_illegal_empty_or_isset_offset(const zval *offset)
+{
+	zend_type_error("Cannot access offset of type %s in isset or empty", zend_get_type_by_const(Z_TYPE_P(offset)));
+}
+
+static void spl_array_illegal_unset_offset(const zval *offset)
+{
+	zend_type_error("Cannot access offset of type %s on unset", zend_get_type_by_const(Z_TYPE_P(offset)));
```

Comment:
Same typo as before: `on unset` -> `in unset`

Reply:
Thanks for the correction!